File: demo_windsim.py
from cplambda_demo import cplambda_demo
from powercurve_demo import powercurve_demo
from BEM_demo import BEM_demo

## written by Dennis van Dommelen, 2014

menu = {}
menu['1']="cp-lambda" 
menu['2']="power curve"
menu['3']="BEM"
menu['4']="exit demo"
while True: 
    options=menu.keys()
    options.sort()
    print 'Choose a demo: \n'
    for entry in options: 
      print entry, menu[entry]

    selection=raw_input("Please Select: ") 
    if selection =='1': 
        cplambda_demo()    
    elif selection == '2': 
        powercurve_demo() 
    elif selection == '3':
        BEM_demo()
    elif selection == '4':
        break
      
    else: 
      print "Unknown Option Selected!" 

File: demo_windsim.m
function demo_windsim

K = MENU('Choose a demo','cp-lambda','power curve','BEM','cp-lambda (Matlab 6.0 version)','power curve (Matlab 6.0 version)','BEM (Matlab 6.0 version)');

if K==1
    playshow cplambda_demo
elseif K==2
    playshow powercurve_demo
elseif K==3
    playshow bem_demo
elseif K==4
    playshow cplambda_demo60
elseif K==5
    playshow powercurve_demo60
else
    playshow bem_demo60
end


